rad/prelude/error-messages.rad

```diff
@@ -309,3 +310,20 @@ Do `rad key create` to create your own key pair and rerun your command."])
 
 (:test "rad-ipfs-name-publish-failure"
   [(rad-ipfs-name-publish-failure "stderr") ==>  "error: Failed to publish key for Git IPFS repository:\nstderr"])
+
+(def process-exit-error
+  "Printed when the `rad ipfs name publish` command in
+  `init-git-ipfs-repo` in `rad-project` fails. Takes stderr of the
+  command as an argument."
+  (fn [command args exit-code stderr]
+    (def command-line (intercalate " " (cons command args)))
+    (def message (string-append "error: Command \"" command-line "\" exited with status code " (show exit-code)))
+    (def stderr-output (map (fn [l] (string-append "  stderr: " l)) stderr))
+    (unlines (cons message stderr-output))
+    ))
+
+(:test "process-exit-error"
+  [(process-exit-error "foo" ["arg1" "arg2"] 5 ["fail1" "fail2"]) ==>
+"error: Command \"foo arg1 arg2\" exited with status code 5
+  stderr: fail1
+  stderr: fail2"])
```

Comment:
For better organization, add a comment as a header for this messages. Currently it is placed under `;; key management`